docker/php/apache.conf

```diff
@@ -0,0 +1,48 @@
+# Source: https://docs.sulu.io/en/latest/cookbook/web-server/apache.html
```

Comment:
@alexander-schranz Do we need to adjust this configuration for development?

Reply:
I would keep it this way.

Reply:
👍